bitrise.yml

```diff
@@ -181,10 +181,15 @@ workflows:
         inputs:
         - notify_email_list: $NOTIFY_EMAILS
     - slack@3:
-        is_always_run: false
         inputs:
         - channel: '#tm-mobile-builds'
         - text: iOS Build Succeeded!
+        - webhook_url_on_error: $SLACK_IOS_WEBHOOK
+        - channel_on_error: '#tm-mobile'
+        - text_on_error: iOS Build Failed!
```

Comment:
I would suggest to add `@mobile-caretaker`

Reply:
fixed - 437446f765dc0d2068b01c7b3429ebe9c1f5fa98